docs/user-guide/reduction-workflow-guidelines.md

```diff
@@ -122,6 +122,31 @@ Users should not have to worry about the concrete type of parameters.
   E.g., `sc.scalar(2, unit='m')` and `sc.scalar(2.0, unit='m')` should be usable interchangeably.
   This can also apply to arrays, for instance, `sc.linspace` and `sc.arange` should be interchangeable but the latter may result in integers while the former typically produces floats.
 
+### C.5: Use a conventional pattern minimizing programming jargon, favor fewer and concepts
+
+**Reason**
+- Terms such as provider or pipeline mean nothing to users, using their language instead reduces cognitive load.
+- Manipulating multiple concepts such as (1) a dict of parameters, (2) a `sciline.Pipeline`, and (3) a `sciline.TaskGraph` is confusing, especially for non-programmers.
+
+**Notes**
+
+- Add one or more `make_workflow` function(s) that returns a `sciline.Pipeline` object, configured with default providers and parameters.
+  - Prefix with `make` and suffix with `workflow`, qualifiers can be added in between, for example `make_abc_workflow`.
+- Avoid creating parameter dicts in notebooks, set parameters on the workflow object directly.
+- Avoid calling `workflow.get` (which would return a `sciline.TaskGraph`), instead call `workflow.compute` and `workflow.visualize`, even if it means listing the result and building the task graph multiple times.
+
+**Example**
+
+```python
+from ess import loki
+
+workflow = loki.make_workflow()
```

Comment:
I am tempted to change this to a camel-case function name (`loki.LokiWorkflow()` or sth. like that). Wouldn't it make it easier to switch to a class later, without having to touch all the notebook?

Reply:
@jl-wynen Thoughts on this one?

Reply:
You don't have to touch the notebooks. It would still be possible to have a function that instantiates the class. Then there would be two ways of doing the same thing which would not be great. But it wouldn't be a big issue either, IMHO.

Reply:
Ok, then let us leave it as it is.